src/docs/product/performance/mobile-vitals.mdx

```diff
@@ -0,0 +1,41 @@
+
+Mobile Vitals are a set of metrics to measure how fast your app starts, the duration of HTTP requests, the number of slow and frozen frames, and more. Each metric provides insights into the overall performance of your mobile application. These metrics give you the details you need to not only prioritize critical performance issues, but to trace issues down to the root cause to solve them faster.
+
+Mobile Vitals are summarized in several graphs on the **Performance** page in [sentry.io](https://sentry.io) when you select "Mobile" in the "Display" dropdown.
+
+![Performance page in Sentry with Mobile Vitals.](mobile-vitals.png)
+
+## App Start
+
+App start metrics track how long your mobile application takes to launch. For this, Sentry measures *cold starts* and *warm starts*.
+
+- **Cold Start**: A cold start refers to when the app launches for the first time, after a reboot or update. The app is not in memory and no process exists.
+- **Warm Start**: A warm start refers to when the app has already launched at least once and is partially in memory. For instance, the user backs out of your app, but then re-launches it. The process may have continued to run, but the app must recreate the activity from scratch.
+
+On iOS, Apple recommends your app take at most 400ms to render the first frame. On Android, the [Google Play console](https://developer.android.com/topic/performance/vitals/launch-time#av) warns you when a cold start takes longer than five seconds or a warm start longer than two seconds. The exact definitions differ slightly per platform. Learn more in our [iOS](/platforms/apple/guides/ios/performance/instrumentation/automatic-instrumentation/) and [Android](/platforms/android/performance/instrumentation/automatic-instrumentation/) SDK docs.
+
+The detail view of a transaction displays the warm start measurement in the right sidebar.
+
+![The event detail of a transaction with a warm start measurement.](app-start-transaction.png)
+
+While the SDKs differentiate between a cold and a warm start, they don't track hot starts or resumes.
+
+## Slow and Frozen Frames
+
+To track the responsiveness of the UI, Sentry measures *slow frames* and *frozen frames*.  Typically, a phone or tablet renders 60 frames per second (fps). At 60 fps, every frame has 16 or 16.67 ms to render.
+
+<!-- TO DO - Find out if there's a formula we can use to explain this vs an example and if that can be used to more generally explain slow and frozen. Is 700ms a universal standard or one platform standard? -->
+
+- **Slow Frames**: Using 60 fps, slow frames are frames that take more than 16 ms (Android) or 16.67 ms (iOS) to render.
+- **Frozen Frames**: Frozen frames are frames that take longer than 700 ms to render.
+
+The frame rate can also be higher, especially as 120 fps displays are becoming more popular, which will change the associated slow frame measurement. This is reflected in the graphs on the **Performance** page when the "Display" selection is "Mobile".
```

Comment:
I think on Android, we currently always go with 60 fps, right @marandaneto? On Apple, we detect the frame rate and adjust the calculation accordingly, but Sentry won't show it to you anywhere, I think.

Reply:
Can confirm that Sentry does not show anything regarding the frame rate, only the slow/frozen/total frames.

Reply:
I was trying to get at the fact that what's considered a slow frame is dependent on the frame rate but I can see how it might seem like I'm implying that the frame rate will be shown in the UI. What about if a reword this way? 

```suggestion
For Apple, the frame rate can be higher, especially as 120 fps displays are becoming more popular. For these apps, Sentry detects the frame rate and adjusts the slow frame calculation accordingly. For Android, the frame rate is always 60 fps. 
```

Reply:
sounds good, I'd remove the Android bits from this sentence though:

> For Android, the slow frames calculation is always based on frame rate of 60 fps. 

The sentence already starts with `For Apple`, so I don't see the need of repeating it, but your call :)